test/pathforaSpec.js

```diff
@@ -1276,7 +1276,7 @@ describe('API', function () {
 
   it('should not show widget if hideAfterAction duration not met', function () {
     var widgetId = 'hideAfterActionWidget1';
-    pathfora.utils.saveCookie('PathforaClosed_' + widgetId, "1," + Date.now());
+    pathfora.utils.saveCookie('PathforaClosed_' + widgetId, "1_" + Date.now());
```

Comment:
we should also add tests to test for the pre-existing cookie with comma case with comments on recommended removal date.